test/integration/logbucket-org/logbucket_org_test.go

```diff
@@ -51,6 +52,9 @@ func TestLogBucketOrgModule(t *testing.T) {
 		assert.Equal(logSinkDestination, logSinkDetails.Get("destination").String(), "log sink destination should match")
 		assert.Equal("resource.type = gce_instance", logSinkDetails.Get("filter").String(), "log sink filter should match")
 
+		//assert writer id has the bucketWriter role
+		logSinkServiceAccount := gcloud.Runf(t, fmt.Sprintf("projects get-iam-policy %s --flatten bindings --filter bindings.role:roles/logging.bucketWriter", projectId))
```

Comment:
nit: [runf](https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test@v0.1.0/pkg/gcloud#Runf) takes in a format string so this can be simplified as 
```suggestion
		logSinkServiceAccount := gcloud.Runf(t, "projects get-iam-policy %s --flatten bindings --filter bindings.role:roles/logging.bucketWriter", projectId)
```